target/bin/sedfile

```diff
@@ -11,19 +11,20 @@
 
 set -ueo pipefail
 
-HASHTOOL="sha1sum"
+function __usage { echo "Usage: ${0} -i <replace/delete operation> <file>" ; }
```

Comment:
Why introducing a function, that is only called once?

Reply:
Probably the intention was to just be consistent with other CLI utility scripts in this folder? That's not such a bad thing even if it might be redundant in this particular file :sweat_smile:

Reply:
All the changes are done for uniformity with the other scripts. In https://github.com/docker-mailserver/docker-mailserver/pull/2493#discussion_r830605532 you pledged for a "continuous look and feel" - this is it (looking at all the other scripts). I have become a bit tired of never-ending small nitpicks like this to be frank. Therefore, I will leave this in (because it really is a nice addition to script uniformity, but I will revert some of the other changes to `sedfile` - otherwise I fear, these changes might never end up in `master`...

Reply:
> you pledged for a "continuous look and feel" - this is it

Fair point. And with that argumentation, I agree with keeping `_usage`. For the same reason I wonder, why you decline to introduce `_log` in this PR? Technically it's no problem to copy the needed files before the first usage of `sedfile`.
Keeping the echos is clearly against a "continuous look and feel".

Reply:
If you want to add `_log` to `sedfile` do it as a follow-up PR, lets just get this stuff merged while it's in a good state and approved :sweat_smile:

Reply:
I agree with @polarathene :) @casperklein I will add a smaller follow-up PR that introduces `_log` to `sedfile` :D